src/test/groovy/org/prebid/server/functional/tests/privacy/GppTransmitTidActivitiesSpec.groovy

```diff
@@ -165,17 +174,23 @@ class GppTransmitTidActivitiesSpec extends PrivacyBaseSpec {
             bidderRequest.source.tid == bidRequest.source.tid
         }
 
+        and: "Don’t affect source.schain if it was present"
+        assert bidderRequest.source.schain == sourceWithSchainAndTid.schain
+
         where:
         createTid << [true, null]
     }
 
-    def "PBS auction should remove bidRequest.(source/imp[0].ext).tid when ext.prebid.createTids=null and activity disallowed"() {
-        given: "Bid requests with TID fields and account id"
+    def "PBS auction should remove bidRequest.(source/imp[0].ext).tid but don't change schain in request when ext.prebid.createTids=null and activity disallowed"() {
```

Comment:
`PBS auction should remove bidRequest.(source/imp[0].ext).tid and don't change schain in request when ext.prebid.createTids=null and activity disallowed`

Please check other places.

Reply:
updated